src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -336,6 +344,7 @@ class BasicSettings extends React.Component {
             items={clusterList}
             selected={data.clusterId || '_'}
             onChange={selectedId => this.handleChange('clusterId', selectedId)}
+            validationState={indicators.cluster}
```

Comment:
What's `indicators.cluster` right after opening Create VM Wizard?

`indicators.cluster = !indicators.name && 'error'` really returns `false` (boolean value) when I open a Create VM Wizard. I would suggest to check other "indicators" as well.

Reply:
Thanks, I missed it, now it's been fixed